geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/DestroyJndiBindingCommand.java

```diff
@@ -49,13 +50,17 @@
       "Destroy a jndi binding that holds the configuration for the XA datasource.";
   static final String JNDI_NAME = "name";
   static final String JNDI_NAME__HELP = "Name of the binding to be destroyed.";
+  static final String IFEXISTS_HELP =
```

Comment:
As above, I'd be explicit about raising an error.

Reply:
Changed it to be -> "Skip the destroy operation when a jndi binding with the same name does not exists. Without specifying this option, this command execution results into an error."